### PR TITLE
Run init task before updateing service

### PR DESCRIPTION
**Pull request by chiragrajk**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by amazreech** [comment]
Hi @chiragrajk,

Thank you for your contribution, apologies on the delay. We will be working to review the changes in the Pull Request. In the meantime please ensure that below steps, if not already completed, are taken care of in your Pull Request:

1. Verify if PR follows semantic pull request [conventions](https://www.conventionalcommits.org/en/v1.0.0/#specification).
Example: Update the PR title to follow conventions feat: add ad-hoc task runs ...

2. Resolve any conflicts on the PR

3. Please run npm run package command to update dist/ folder with latest dependencies.

4. Please update PR descirption with any additional context on the code. 

Thank you!

File: index.test.js [hunk 1]
```diff
@@ -149,6 +153,57 @@ describe('Deploy to ECS', () => {
                 }
             };
         });
+
+        mockRunTasks.mockImplementation(() => {
+            return {
+                promise() {
```
Comment (trivikr) [inline index.test.js:159]: The mock needs to be updated as the action is now using JS SDK v3

Refer: https://github.com/aws-actions/amazon-ecs-deploy-task-definition/pull/529

**Review by trivikr — COMMENTED** [review]
Provided diff for transforming JS SDK v2 APIs to v3.

Refs: https://github.com/aws-actions/amazon-ecs-deploy-task-definition/pull/529

**Comment by amazreech** [comment]
Changes to run init task has been merged via [PR #340](https://github.com/aws-actions/amazon-ecs-deploy-task-definition/pull/304). Closing out this PR.

**Pull request closed without merging**